app.js

```diff
@@ -76,6 +80,8 @@ if (app.get('env') == 'development') {
   app.use(errorHandler());
 }
 
+const c = "vuln c"
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in password_token.

<h6>
<a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 562</a> | Line 2</h6>

<details>

<summary><b>Data flow: 3 steps</b></summary>
<br/>

[app.js#L47 (SOURCE)](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

[app.js#L69](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L69)

[app.js#L83 (SINK)](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L83)

</details>